package.json

```diff
@@ -55,7 +56,10 @@
         "sweetalert": "^2.1.2",
         "three": "npm:@exabyte-io/three@2023.6.22-0",
         "underscore": "^1.8.3",
-        "underscore.string": "^3.3.4"
+        "underscore.string": "^3.3.4",
+        "@exabyte-io/cove.js": "2023.7.14-0",
+        "@mui/lab": "^5.0.0-alpha.120",
```

Comment:
Why do we need this?

Reply:
It was a solution to import cove.js, from error messages those libraries were missing.

Reply:
@zoomchik - do we need this? Any way to avoid?

Reply:
@timurbazhirov We need this packages as peerDependencies at least to avoid multiple copies of React. https://legacy.reactjs.org/warnings/invalid-hook-call-warning.html#duplicate-react